package.json

```diff
@@ -14,7 +14,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "fix-js": "prettier --write \"./**/*.{js,jsx}\"",
+    "fix-js": "prettier --write \"src/**/*.{js,jsx}\"",
```

Comment:
fix:js : "prettier --write \"src/**/*.{js,jsx}\"",
fix:styles: "prettier --write \"src/**/*.{css,scss}\"",

and please add aditional script that will fix both together 

npm run fix:   fix:js && fix:styles 

https://stackoverflow.com/questions/39172536/running-npm-scripts-sequentially  

Also please sort scrpits alphabetically